data/virt_autotest/guest_params_xml_files/sles_15_sp4_64_xen_hvm_uefi_without_power_management_x86_64_using_virtmanagerlt3.0.0.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp4</guest_version>
+    <guest_version_major>15</guest_version_major>
+    <guest_version_minor>4</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build></guest_build>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>xen</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>1024,maxmemory=4096</guest_memory>
+    <guest_vcpus>1,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel></guest_cpumodel>
+    <guest_metadata></guest_metadata>
+    <guest_xpath></guest_xpath>
+    <guest_installation_automation>autoyast</guest_installation_automation>
+    <guest_installation_automation_file>sles_15_plus_xen_hvm_guest_graphical_x86_64.xml</guest_installation_automation_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1#rd.debug#rd.memdebug=4#rd.udev.debug#display=3#ipv4only=1</guest_installation_extra_args>
```

Comment:
guest uses different display device instead of normal autoconsole VNC?

Reply:
Display should be explicitly given to start installation, otherwise it can not be initialized.